client/widgets/CGarrisonInt.cpp

```diff
@@ -514,11 +514,14 @@ void CGarrisonInt::setSplittingMode(bool on)
 
 	if (inSplittingMode || on)
 	{
-		for(CGarrisonSlot * slot : slotsUp)
-			slot->setHighlight( ( on && (slot->our() || slot->ally()) && (slot->creature == nullptr || slot->creature == getSelection()->creature)));
-
-		for(CGarrisonSlot * slot : slotsDown)
-			slot->setHighlight( ( on && (slot->our() || slot->ally()) && (slot->creature == nullptr || slot->creature == getSelection()->creature)));
+		for(CGarrisonSlot * slot : slotsUp){
```

Comment:
No braces at the end of line, please. http://wiki.vcmi.eu/index.php?title=Coding_guidelines#Where_to_put_braces